src/app.ts

```diff
@@ -70,6 +71,19 @@ const sessionStore: SessionStore | undefined =
 
 const app = express();
 
+const protectConfig: ProtectionConfig = {
+  production: process.env.NODE_ENV === "production",
+  clientRetrySecs: 1,
+  sampleInterval: 5,
+  maxEventLoopDelay: 400,
+  maxHeapUsedBytes: 0,
+  maxRssBytes: 0,
+  errorPropagationMode: false,
+  logging: "info"
```

Comment:
There's an extra `logStatsOnReq` that I didn't add here. It was failing bc of type errors (for some reason it could only be `false | undefined`. Not sure how to resolve this other than maybe declaring a custom type file?

Reply:
There is also a warning with setting [this config to true](https://www.npmjs.com/package/overload-protection#logstatsonreq-false) that it could potentially increase pressure on the event loop

Reply:
I don't think we should set `logStatsOnReq` - that would be very noisy and limited use. The `frontend-vital-signs` thing should capture relevant metrics instead 🤞 

(If you do need to override the library type definitions there are a couple of options depending on how it's been implemented - mainly through 'declaration merging'. You can see a couple of examples at the top of `app.ts`)